stl-decomp-4j/pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.github.servicenow.stl4j</groupId>
   <artifactId>stl-decomp-4j</artifactId>
   <packaging>jar</packaging>
-  <version>1.0.4-RC</version>
+  <version>1.0.5-SNAPSHOT</version>
```

Comment:
Why designate snapshot instead of RC?

Reply:
RC is the designation I give it once I've created a release branch - should always be SNAPSHOT on master. The tag builds are supposed to automatically remove whatever annotation is after the number.

Reply:
I'd be happy to walk you through the release process - would be nice for someone else to be familiar with it. :)